components/gitpod-protocol/src/protocol.ts

```diff
@@ -254,6 +254,9 @@ export interface AdditionalUserData {
     // whether the user has been migrated to team attribution.
     // a corresponding feature flag (team_only_attribution) triggers the migration.
     isMigratedToTeamOnlyAttribution?: boolean;
+    // user globol workspace timeout
+    workspaceTimeout?: string;
+    disabledClosedTimeout?: boolean;
```

Comment:
Is this a typo maybe and should be `disabledCloseTimeout` instead?
Also, what exactly does it control (a comment would be golden).